.travis.yml

```diff
@@ -38,6 +38,9 @@ jobs:
                 openjdk:8-jdk-alpine \
                 ./gradlew --no-daemon testcontainers:test --tests '*GenericContainerRuleTest'
 
+    - env: [ NAME="examples"]
```

Comment:
WDYT about adding it to CircleCI instead?
Otherwise we will be running it during the release (while depending on SNAPSHOT from Jitpack)

Reply:
Sure, was even thinking about adding it to both.
But your reasoning makes sense, so I switch.